src/injected/analyzers/tab-stops-orchestrator.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { WindowUtils } from 'common/window-utils';
+import { AllFrameRunnerTarget } from 'injected/all-frame-runner';
+import {
+    TabStopRequirementResult,
+    TabStopsRequirementEvaluator,
+} from 'injected/tab-stops-requirement-evaluator';
+import { TabbableElementGetter } from 'injected/tabbable-element-getter';
+import { FocusableElement } from 'tabbable';
+
+export class TabStopRequirementOrchestrator
+    implements AllFrameRunnerTarget<TabStopRequirementResult>
+{
+    public readonly name: string = 'TabStopRequirementOrchestrator';
```

Comment:
What will this be used for?

Reply:
my fault - usage of 'name' is in all-frame-runner from https://github.com/microsoft/accessibility-insights-web/pull/5036. We need the target to supply a suffix to frame-messenger commands since command strings should be globally unique. Discussed offline & will update this field in a follow-up PR.